crates/iota-indexer/src/backfill/mod.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Mysten Labs, Inc.
+// Modifications Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{ops::RangeInclusive, sync::Arc};
+
+use async_trait::async_trait;
+use clap::{Subcommand, ValueEnum};
+
+use crate::{backfill::sql::sql_backfill::SqlBackfill, db::ConnectionPool, errors::IndexerError};
+
+pub(crate) mod ingestion;
+pub mod runner;
+pub(crate) mod sql;
+
+/// Encapsulates the logic to process and persist data for a chunk of
+/// checkpoints.
+#[async_trait]
+pub(crate) trait Backfill: Send + Sync {
+    async fn backfill_range(
+        &self,
+        pool: ConnectionPool,
+        range: &RangeInclusive<usize>,
+    ) -> Result<(), IndexerError>;
+}
+
+// Subcommands for selecting a backfill task to run.
+// Each variant corresponds to a different backfill implementation.
```

Comment:
Use `///` instead of `//`.

Reply:
Fixed in https://github.com/iotaledger/iota/pull/7603